spec/rspec/rails/example/view_example_group_spec.rb

```diff
@@ -225,6 +235,7 @@ def controller
 
       it 'is accessible to hooks' do
         with_isolated_config do
+          view = double("view")
```

Comment:
No idea why it was passing before.

Reply:
Is `view` not also a method?

Reply:
Doesn't seem so, at least not it the bare context of `ExampleGroup.describe`.
This is needed [here](https://github.com/rspec/rspec-rails/pull/2264/files/87aee32e014aed4c818156a9c473c5de8682b8cc#diff-0cdf276129b0b83c6314283584c4df6eR241).
Picked it up [here](https://github.com/rspec/rspec-rails/pull/2264/files/87aee32e014aed4c818156a9c473c5de8682b8cc#diff-0cdf276129b0b83c6314283584c4df6eR231).
I suspect that `with_isolated_config` was swallowing `NoMethodError` previously.

Reply:
Nope nor these links

Reply:
`with_isolated_config` does not swallow errors, it just runs an `ensure`

Reply:
It's defined here: `lib/rspec/rails/example/view_example_group.rb:81` so this shouldn't be removed...

Reply:
You need to keep `with_isolated_config`, if your changes here are causing an error you need to ensure the right methods are defined, doing `double` here is just wrong..

Reply:
A call to `allow(view).to receive(:a_stubbed_helper) { :value }` results in:
```
RSpec::Mocks::MockExpectationError: #<#<Class:0x00007fa59165da18>:0x00007fa59165ceb0 @_config={}, ...> does not implement: a_stubbed_helper
```
that for some reason doesn't bubble up.
The same happens if you move stubbing directly to the example:
```ruby
          group = RSpec::Core::ExampleGroup.describe 'a view', :type => :view do
            specify {
              allow(view).to receive(:a_stubbed_helper) { :value }
              run_count += 1
            }
          end
```
Example returns prematurely, `run_count` is not incremented, but there's no exception from mocks. Is it a bug?

Changed to stub an existing `render` method instead.

Removed `NullObject.new` from `group.run`'s arguments, it doesn't seem to affect this in any way.